pkg/Library/App/surface-imports.js

```diff
@@ -8,7 +8,8 @@ import '../DevTools/Dom/surface-walker.js';
 import '../DevTools/Dom/resource-view.js';
 import '../D3/d3p/visual-arc/data-graph.js';
 import '../Goog/good-map/good-map.js';
-import '../Media/image-resource.js';
+//import '../Media/image-resource.js';
+import '../NewMedia/image-resource.js';
```

Comment:
just curios - do we plan to eventually get rid of `Media/` and replace it with `NewMedia/`?

Reply:
Yes